PeopleAndAppleStockPrices/PeopleAndAppleStockPrices/Detail View Controllers/PricesDetailViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PricesDetailViewController.swift
+//  PeopleAndAppleStockPrices
+//
+//  Created by Levi Davis on 9/4/19.
+//  Copyright © 2019 Pursuit. All rights reserved.
+//
+
+import UIKit
+
+class PricesDetailViewController: UIViewController {
+    
+    var price: Stock!
+
+    @IBOutlet weak var someImageOutlet: UIImageView!
+    @IBOutlet weak var dateLabelOutlet: UILabel!
+    @IBOutlet weak var openingPriceLabelOutlet: UILabel!
+    @IBOutlet weak var closingPriceLabelOutlet: UILabel!
+    
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        // Do any additional setup after loading the view.
+    }
+    
+
+    private func setupView() {
+        selectImage()
+        dateLabelOutlet.text = price.date
+        openingPriceLabelOutlet.text = "\(price.open)"
+        closingPriceLabelOutlet.text = "\(price.close)"
+    }
+
+    private func selectImage() {
```

Comment:
This is solid, but maybe do this logic in the model? Then select the image depending on the output.